COMPONENT-API-OLD.md

```diff
@@ -0,0 +1,376 @@
+# Component API
```

Comment:
I'd just recommend removing this file entirely to reduce any confusion. We can always recover content from git.